lib/livebook/fly_api.ex

```diff
@@ -193,11 +193,17 @@ defmodule Livebook.FlyAPI do
   """
   @spec await_machine_started(String.t(), String.t(), String.t()) :: :ok | {:error, error}
   def await_machine_started(token, app_name, machine_id) do
+    # The maximum supported timeout is 60s, but the machine may take
+    # longer to start if it uses a large Docker image (such as CUDA),
+    # provided the image is not already in the Fly cache. To achieve
+    # a longer wait, we retry request timeouts.
     with {:ok, _data} <-
            flaps_request(token, "/v1/apps/#{app_name}/machines/#{machine_id}/wait",
              params: %{state: "started", timeout: 60},
              receive_timeout: 90_000,
-             retry: false
+             retry: fn _req, result -> match?(%Req.Response{status: 408}, result) end,
```

Comment:
408s are retried by default so you can use `retry: :transient` to additionally retry on network errors. You could even leave `retry: ` out since Req retries by default though I'm thinking about changing that default for Req v1.0 (https://github.com/wojtekmach/req/issues/383)

Reply:
Oh, I was aware that this is the default, just wanted to be explicit here, since it has a very specific purpose. But yeah, I guess there is no reason to not retry the network errors too as usual; the request is long-running network errors are actually more likely. I will change to explicit `retry: :safe_transient`.

Regarding the default, I see arguments for both and I don't have a strong opinion, but I am leaning towards opt-in being a more intuitive choice.